src/features/company/components/Company.tsx

```diff
@@ -46,7 +46,7 @@ export const Company = () => {
         loaded={selectedCompanyOverview !== undefined}
         render={renderCompanyComponent}
       />
-      <Peers peers={selectedTopPeers} />
+      <Peers peers={selectedTopPeers && selectedTopPeers} />
```

Comment:
The added `&& selectedTopPeers` makes no difference here

Reply:
Good spot, will remove in a new PR